tests/cases/fourslash/autoImportTypeImport1.ts

```diff
@@ -0,0 +1,41 @@
+/// <reference path="fourslash.ts" />
+
+// @verbatimModuleSyntax: true
+// @target: esnext
+
+// @Filename: /foo.ts
+//// export const A = 1;
+//// export type B = { x: number };
+//// export type C = 1;
+//// export class D = { y: string };
+
+// @Filename: /test.ts
+//// import { A, D, type C } from './foo';
+//// const b: B/**/ | C;
+//// console.log(A, D);
+
+goTo.marker("");
+
+verify.importFixAtPosition([
+`import { A, D, type C, type B } from './foo';
+const b: B | C;
+console.log(A, D);`],
+    /*errorCode*/ undefined,
+    { organizeImportsTypeOrder: "inline" }
+);
+
+verify.importFixAtPosition([
+`import { A, D, type B, type C } from './foo';
+const b: B | C;
+console.log(A, D);`],
+    /*errorCode*/ undefined,
+    { organizeImportsTypeOrder: "last" }
+);
+
+verify.importFixAtPosition([
+`import { A, D, type C, type B } from './foo';

```

Comment:
Ah, this is "first", so it's unsorted. Gotcha. I feel like these would benefit from a comment on each saying which are unsorted and which are sorted, but maybe those would go out of date...

Reply:
I'll work on adding some comments since it sounds like a good idea to make less people confused in the future. If the comments go out of date, then the tests will break (and hopefully whoever breaks the tests also updates the comments :0 )